domain/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+</manifest>
```

Comment:
The `domain` layer should be a `pure Kotlin module`. It should not have any `Android` related dependencies or files.